extensions/README.md

```diff
@@ -16,17 +16,15 @@ Linux - LibSecret is used for encryption.
 
 ## Building
 
-The extensions contain prebuild binaries. To build from source, you will need Python on you path.
+The extensions contain prebuild binaries. To build from source, you will need Python on you path,
+as [node-gyp](https://github.com/nodejs/node-gyp) is used to build addons for accessing system APIs.
+
+On linux, the library uses `libsecret` so you may need to install it.
```

Comment:
I would include or link to instructions for how to check that it is installed and installed it if not.

Reply:
Added instructions for a couple of distributions.